packages/common/tests/test_resources.py

```diff
@@ -1,44 +1,47 @@
-from subprocess import run
 from pathlib import Path
+from subprocess import run
 
-from pytest import mark
-from dagster import build_init_resource_context
-import docker
-
+import pytest
 from bag3d.common import resources
+from dagster import build_init_resource_context
 
 
-@mark.parametrize(
-    ("config", "local_path"),
+@pytest.mark.parametrize(
+    ("config", "data_dir", "filename"),
     (
-            ({"docker": {"image": "osgeo/gdal:alpine-small-3.5.2",
-                         "mount_point": "/tmp"}}, Path("/data/top10nl.zip")),
-            ({"exes": {"ogrinfo": "ogrinfo"}}, Path("/data/top10nl.zip"))
```

Comment:
Here I changed this so that the path to the data is not hardcoded but it depends on where the data has been downloaded by the user.